kafka/producer/base.py

```diff
@@ -75,8 +79,8 @@ def _send_upstream(queue, client, codec, batch_time, batch_size,
             client.send_produce_request(reqs,
                                         acks=req_acks,
                                         timeout=ack_timeout)
-        except Exception:
-            log.exception("Unable to send message")
+        except:
+            log.exception('Failed to send messages to Kafka. Batch size: {0}. Topics: {1}'.format(len(reqs), topics))
```

Comment:
I thought I had left a comment on this. I would revert this to reduce the diff although this message is better.